### ISSUE TITLE
dropout(): argument 'input' (position 1) must be Tensor, not str

### ISSUE THREAD

**Issue by encaion** [SKTBrain/KoBERT#69]
안녕하세요. 다음의 코드와 같이 기존에 사용하던 koBERT에서 #67 를 참고하여 다음의 코드를 사용하여 설치를 진행하였습니다.

```
!pip install 'git+https://github.com/SKTBrain/KoBERT.git#egg=kobert_tokenizer&subdirectory=kobert_hf'
```

하기와 같이 조건을 맞추었으며,

> Python >= 3.6
> PyTorch >= 1.8.1
> transformers >= 4.8.2
> sentencepiece >= 0.1.91

작업을 위한 라이브러리 설치  코드는 다음과 같습니다.
```
!pip install gluonnlp==0.10.0 pandas tqdm   
!pip install mxnet
!pip install sentencepiece==0.1.91
!pip install torch==1.8.1
!pip install torchtext==0.9.1 # Mar 26, 2021
!pip install torchvision==0.9.1 # Mar 26, 2021
!pip install torchaudio==0.8.1 # Mar 26, 2021
!pip install transformers==4.8.2
!pip install 'git+https://github.com/SKTBrain/KoBERT.git#egg=kobert_tokenizer&subdirectory=kobert_hf'
```

그런데 #40 (sequence_output, pooled_output이 torch.size가 아닌 str 타입으로 출력)와 같이 에러가 발생하였습니다.
```
dropout(): argument 'input' (position 1) must be Tensor, not str
```

transformers 버전과 torch 버전을 문서에 나온 권장 버전의 범위에서 다양한 조합으로 시도하였으나 제대로 학습이 되지 않는 문제가 발생합니다.

--------------------------------

어쩌다보니 issue 등록 중에 해결하였습니다만, `BertModel.from_pretrained()` 에서 `return_dict=False` 를 해주니 제대로 학습이 되었습니다.
```
def get_kobert_model(model_path, vocab_file, ctx="cpu"):
    # bertmodel = BertModel.from_pretrained(model_path)
    bertmodel = BertModel.from_pretrained(model_path, return_dict=False)
    device = torch.device(ctx)
    bertmodel.to(device)
    bertmodel.eval()
    vocab_b_obj = nlp.vocab.BERTVocab.from_sentencepiece(vocab_file,
                                                         padding_token='[PAD]')
    return bertmodel, vocab_b_obj

from kobert_tokenizer import KoBERTTokenizer
tokenizer = KoBERTTokenizer.from_pretrained('skt/kobert-base-v1')
bertmodel, vocab = get_kobert_model('skt/kobert-base-v1',tokenizer.vocab_file, ctx = "cuda")
```

이슈는 해결하였으나 조금이나마 기여가 되었으면 하는 바람에 남겨봅니다.

언제나 좋은 자료 제공 감사합니다.

**Comment by bage79** [comment]
@encaion 상세한 설명 감사드립니다. 👍 
코드에 반영하였습니다. 
https://github.com/SKTBrain/KoBERT/commit/47eb1b5f82d59cb1bf3fcf723b2e747480cfa859
